### PR TITLE
aria2: stub, stub test, improved tests and meta.yml

### PR DESCRIPTION
**Pull request by leoisl**
This PR improves the aria2 module by adding stub, stub test, improved tests and meta.yml.

### CODE REVIEW

File: modules/nf-core/aria2/tests/main.nf.test [hunk 1]
```diff
@@ -11,20 +10,35 @@ nextflow_process {
...
+            }
+        }
         then {
             assertAll(
                 { assert process.success },
-                { assert process.out.downloaded_file != null },
-                { assert process.out.downloaded_file.get(0).get(1) ==~ ".*.bam"  },
-                { assert snapshot(process.out.downloaded_file).match("downloaded_file")},
-                { assert snapshot(process.out.versions).match("versions") }
+                { assert snapshot(file(process.out.downloaded_file[0][1]).name).match() }
```
Comment (mashehu) [inline modules/nf-core/aria2/tests/main.nf.test:41]: you can't do a full snapshot here because of the empty file warning?

Comment (leoisl) [reply modules/nf-core/aria2/tests/main.nf.test:41]: Yes, linting fails with:
```
md5sum for empty file found: d41d8cd98f00b204e9800998ecf8427e 
```

I am then just snapshotting the name of the file as described in https://hackmd.io/@nf-core/ryWsFLwyT#Snapshot-Only-the-Filename

Happy to switch this to a better way of asserting stub test output, if you have one!

Comment (mashehu) [reply modules/nf-core/aria2/tests/main.nf.test:41]: we are working on a fix 🙂 https://github.com/nf-core/tools/pull/2571#pullrequestreview-1776967107

Comment (leoisl) [reply modules/nf-core/aria2/tests/main.nf.test:41]: Oh wow, that was quick, thanks a lot! Should we wait this fix to be merged to `nf-core/tools` before merging this PR?

Comment (mashehu) [reply modules/nf-core/aria2/tests/main.nf.test:41]: yes, should be merged by today and then we don't need to come back to this module, if you are not in a hurry

Comment (leoisl) [reply modules/nf-core/aria2/tests/main.nf.test:41]: Not in a hurry, happy to wait!

Comment (leoisl) [reply modules/nf-core/aria2/tests/main.nf.test]: Hey @mashehu just saw the message on slack that `nf-core` linting will no longer fail if an empty file md5sum is found in stub tests. I am now snapshotting the whole process.out even in the stub test (see https://github.com/nf-core/modules/pull/4572/commits/3a5dfe697320d5eb74a72e93a5e80c1974710cf8)

**Review by mashehu — APPROVED** [review]

**Pull request merged**